packages/web/server/api/edge-router.ts

```diff
@@ -17,5 +18,6 @@ export const edgeRouter = createTRPCRouter({
   earn: earnRouter,
   transactions: transactionsRouter,
   chains: chainsRouter,
+  portfolio: portfolioRouter,
```

Comment:
IMO, we should add this to the local router. Simply remove it from here and import and add to local-router.ts file in same folder.

There's no benefit, as far as I can tell, to first send a request to the edge since we don't want to cache user data. Instead, the user's browser will directly send a request to SQS, avoiding a hop through the edge function. 

It should notably increase client side performance.